src/components/files/FileActions/FileActionsContext.tsx

```diff
@@ -109,12 +106,12 @@ export const FileActionsContextProvider = (props: ChildrenProp) => {
   const actions = useMemo(
     () => ({
       rename: (item: FilesActionItem) => renameFile(item),
-      download: (item: FilesActionItem) => void downloadFile(item),
+      download: (item: FilesActionItem) => console.log(item),
```

Comment:
Any suggestions on what to do here @CarsonF ? We've essentially made the `downloadFile` here unnecessary, but it seems like the context and some of the UI still relies on the download action from this context. 

Basically saying I don't think this is "done" until we remove this console log for whatever we want to do here. Might even require some more refactoring.

Reply:
Definitely more refactoring needed at some point. So much of this is head spinning.

I think at this point I'd just put a comment saying it was downloaded via browser / anchor tag.